tools/unitctl/unit-client-rs/src/unit_client.rs

```diff
@@ -185,7 +185,14 @@ impl UnitClient {
     }
 
     pub fn new_http(control_socket: ControlSocket) -> Self {
-        let remote_client = Client::builder().build(HttpsConnector::new());
+        let remote_client = Client::builder()
+            .build(HttpsConnectorBuilder::default()
+                   .with_native_roots()
+                   .unwrap_or_else(|_| HttpsConnectorBuilder::default()
+                                   .with_webpki_roots())
+                   .https_or_http()
+                   .enable_http1()
+                   .wrap_connector(HttpConnector::new()));
```

Comment:
this one apparently is only available with the http2 crate feature: https://docs.rs/hyper-rustls/latest/hyper_rustls/struct.HttpsConnectorBuilder.html#method.wrap_connector-1

The http2 flag isn't enabled by default per https://docs.rs/crate/hyper-rustls/latest/features, and it's also not enabled in the cargo.toml file 🤔 

How does this work? 😄 It built completely fine, but docs say it shouldn't have...?

(this is a rust thing, not a you thing)

Reply:
Oh, there are two `wrap_connector` methods, one is on the `ConnectorBuilder<WantsProcols2>` impl, the other is on the `ConnectorBuilder<WantsProtocols3>` impl.

The second one needs the http2 crate, the first one does not.